pkg/apis/devconsole/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -13,7 +13,7 @@ func (in *Component) DeepCopyInto(out *Component) {
 	*out = *in
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
-	in.Spec.DeepCopyInto(&out.Spec)
```

Comment:
@corinnekrych why do we need to change this ?

Reply:
I use latest version of operator-sdk v0.7.0
what about yours?

Reply:
mine is v0.6.0, I was just wondering because there is a method `DeepCopyInto` on ComponentSpec struct. If we no longer using it anywhere shouldn't that be auto removed ? (not sure if we are using anywhere)